apps/artemis-web/utils/power-actions.ts

```diff
@@ -0,0 +1,111 @@
+import { configure } from 'react-hotkeys';
+
+export const commands = (role) => (
+
+    role === 'admin' ? {
+        Dashboard: {
+            name: 'Dashboard',
+            shortcut: 'alt+d',
+            callback: () => (window.location.pathname = '/dashboard'),
+        },
+        BGPUpdates: {
+            name: 'BGP Updates',
+            shortcut: 'alt+b',
+            callback: () => (window.location.pathname = '/bgpupdates'),
+        },
+        Hijacks: {
+            name: 'Hijacks',
+            shortcut: 'alt+h',
+            callback: () => (window.location.pathname = '/hijacks'),
+        },
+        System: {
+            name: 'System',
+            shortcut: 'alt+s',
+            callback: () => (window.location.pathname = '/admin/system'),
+        },
+        UserManagement: {
+            name: 'User Management',
+            shortcut: 'alt+m',
+            callback: () => (window.location.pathname = '/admin/user_management'),
+        },
+        Password: {
+            name: 'Password Change',
+            shortcut: 'alt+p',
+            callback: () => (window.location.pathname = '/password_change'),
+        },
+        Config: {
+            name: 'Config Comparison',
+            shortcut: 'alt+c',
+            callback: () => (window.location.pathname = '/config_comparison'),
+        },
+    } : {
+        Dashboard: {
+            name: 'Dashboard',
+            shortcut: 'alt+d',
+            callback: () => (window.location.pathname = '/dashboard'),
+        },
+        BGPUpdates: {
+            name: 'BGP Updates',
+            shortcut: 'alt+b',
+            callback: () => (window.location.pathname = '/bgpupdates'),
+        },
+        Hijacks: {
+            name: 'Hijacks',
+            shortcut: 'alt+h',
+            callback: () => (window.location.pathname = '/hijacks'),
+        },
+        Password: {
+            name: 'Password Change',
+            shortcut: 'alt+p',
+            callback: () => (window.location.pathname = '/password_change'),
+        },
+        Config: {
+            name: 'Config Comparison',
+            shortcut: 'alt+c',
+            callback: () => (window.location.pathname = '/config_comparison'),
+        },
+    });
+
+export const keyMap = {
+    TOGGLE_MODAL: 'cmd+k',
```

Comment:
mention here the Mac and Linux variants (see review comment)

Reply:
you mean mention this as code comments?

Reply:
yap, in the keymap description

Reply:
Done :)